src/_actions/paymentActions.js

```diff
@@ -19,7 +19,7 @@ const makePayment = (phone, amount) => async (dispatch) => {
       console.log(response)
     dispatch({ type: MAKE_PAYMENT_SUCCESS, payload: response });
 
-    })
+    }).catch(err=>console.log(err))
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch(err =>console.log(err))
```